### DISCUSSION TITLE
When to choose --start-code?

### DISCUSSION THREAD

**Discussion by clidx** [quietvoid/dovi_tool#184] [Q&A] [answered]
I noticed that `--start-code` was added at some point. What situations would we want to create an RPU with each option? If `annex-b` follows x265 behaviour, is it more compatible or something?

**Comment by quietvoid** [comment, marked as answer]
The only situation would be to have identical output compared to x265, in almost all relevant cases.
So normally using x265 with the RPU would be identical to encoding without RPU, and injecting with `--start-code annex-b`.

**Reply by clidx** [reply]
Right, so it doesn't really have any practical implications for playback. Thanks.
